### PR TITLE
feat: Make theme gallery strings translatable

### PR DESCRIPTION
**Pull request by maraoz**
## Description

Made theme gallery strings translatable

## Motivation and Context
Enables to translate these strings in the future

## How Has This Been Tested?
English and Spanish. Manual tests of Settings page.

## Screenshots (if appropriate):
image
image

### CODE REVIEW

File: public/locales/es/common.json [hunk 1]
```diff
@@ -124,6 +124,11 @@
     "dark": "Mono Dark",
     "explorer": "Explorer",
     "light": "Mono Light",
-    "system-prefs": "Sistema"
+    "system-prefs": "Sistema",
+    "custom": "Personalizado",
```
Comment (turtlemoji) [inline public/locales/es/common.json:128]: I will live themes: {} just for the themes names, and move `custom` and `gallery` keys to `modals.json`.
Something like:
```
"community-modal": {
    "title: ".....",
    "community: "....",
    "custom": "Personalizado"
}
```

Comment (maraoz) [reply public/locales/es/common.json:128]: ok! any action needed on my end?

Comment (turtlemoji) [reply public/locales/es/common.json:128]: @maraoz Can you pull develop and make the change I mentioned, didn't have any time to do it!

Comment (maraoz) [reply public/locales/es/common.json]: @turtlemoji ready for review, I did as you requested!

same as before, I tested this manually locally by switching between English and Spanish. Seems to work

**Review by xgambitox — APPROVED** [review]

**Pull request merged**